src/plugins/kibana_utils/docs/state_containers/README.md

```diff
@@ -0,0 +1,50 @@
+# State containers
```

Comment:
I wonder why we use it `state container` instead of `store`. It seems to me like the same thing, but one is shorter?

Reply:
We just call it "state containers" everywhere so for consistency I cleaned out "store" and replaced all mentions with "state containers".